fluster/decoders/ffmpeg.py

```diff
@@ -56,8 +61,17 @@ def decode(
     ) -> str:
         '''Decodes input_filepath in output_filepath'''
         # pylint: disable=unused-argument
-        cmd = shlex.split(FFMPEG_TPL.format(
-            self.cmd, input_filepath, str(output_format.value), output_filepath))
+        cmd = shlex.split('ffmpeg -version')
+        output = subprocess.check_output(
+            cmd, stderr=subprocess.DEVNULL).decode('utf-8')
+        version = re.search(r"\d\.\d\.\d", output)
```

Comment:
What do you think about put this logic in an external cached method (with `@lru_cache`)?